test-suite-binaries/test.sh

```diff
@@ -61,5 +61,9 @@ echo "RUN elf-ARM64-bash --depth 5" | tee -a  ./test_output
 $RUN --binary ./elf-ARM64-bash --depth 5 1>> ./test_output
 echo "RUN elf-PPC64-bash --depth 5" | tee -a  ./test_output
 $RUN --binary ./elf-PPC64-bash --depth 5 1>> ./test_output
+echo "RUN elf-Linux-RISCV_64 --depth 8" | tee -a  ./test_output
+$RUN --binary ./elf-PPC64-bash --depth 5 1>> ./test_output
+echo "RUN elf-Linux-RISCV_32 --depth 8" | tee -a  ./test_output
+$RUN --binary ./elf-PPC64-bash --depth 5 1>> ./test_output
```

Comment:
It seems that you forgot to replace ppc binary with risv in test.sh

Reply:
fix